lua/starfall/libs_sh/file.lua

```diff
@@ -80,10 +80,9 @@ end
 
 --- Creates a directory
 -- @param path Filepath relative to data/sf_filedata/. Cannot contain '..'
--- @return Error message if applicable
 function file_library.createDir ( path )
 	if not SF.Permissions.check( SF.instance.player, path, "file.write" ) then SF.throw( "Insufficient permissions", 2 ) end
 	SF.CheckType( path, "string" )
-	if path:find( "..", 1, true ) then error( "path contains '..'" ) return end
+	if path:find( "..", 1, true ) then SF.throw( "path contains '..'" ) return end
```

Comment:
Shouldn't there be a 'level' to blame for this error? Similar to line 84 for permissions?